tests/Fixtures/TestBundle/Entity/CompositeRelation.php

```diff
@@ -50,7 +50,7 @@ class CompositeRelation
      */
     public function getId()
     {
-        return sprintf('%s-%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
+        return sprintf('%s;%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
```

Comment:
Yes, is it still a problem if we remove this method from `CompositeRelation` (and from the related tests too, of course)?